x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/transactions/transactions_groups_main_statistics.spec.ts

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { sum } from 'lodash';
+import { LatencyAggregationType } from '@kbn/apm-plugin/common/latency_aggregation_types';
+import { ApmDocumentType, ApmTransactionDocumentType } from '@kbn/apm-plugin/common/document_type';
+import { RollupInterval } from '@kbn/apm-plugin/common/rollup';
+import { apm, timerange } from '@kbn/apm-synthtrace-client';
+import type { ApmSynthtraceEsClient } from '@kbn/apm-synthtrace';
+import type { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
+
+export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const apmApiClient = getService('apmApi');
+  const synthtrace = getService('synthtrace');
+
+  const serviceName = 'synth-go';
+  const start = new Date('2021-01-01T00:00:00.000Z').getTime();
+  const end = new Date('2021-01-01T01:00:00.000Z').getTime() - 1;
+
+  async function callApi(overrides?: {
+    path?: {
+      serviceName?: string;
+    };
+    query?: {
+      start?: string;
+      end?: string;
+      transactionType?: string;
+      environment?: string;
+      kuery?: string;
+      latencyAggregationType?: LatencyAggregationType;
+      documentType?: ApmTransactionDocumentType;
+      rollupInterval?: RollupInterval;
+      useDurationSummary?: boolean;
+    };
+  }) {
+    const response = await apmApiClient.readUser({
+      endpoint: 'GET /internal/apm/services/{serviceName}/transactions/groups/main_statistics',
+      params: {
+        path: { serviceName, ...overrides?.path },
+        query: {
+          start: new Date(start).toISOString(),
+          end: new Date(end).toISOString(),
+          latencyAggregationType: 'avg' as LatencyAggregationType,
+          transactionType: 'request',
+          environment: 'ENVIRONMENT_ALL',
+          useDurationSummary: false,
+          kuery: '',
+          documentType: ApmDocumentType.TransactionMetric,
+          rollupInterval: RollupInterval.OneMinute,
+          ...overrides?.query,
+        },
+      },
+    });
+    expect(response.status).to.be(200);
+    return response.body;
+  }
+
+  describe('Transaction groups main statistics', () => {
+    describe('Transaction groups main statistics when data is not loaded', () => {
```

Comment:
Nit: the text is repeating from the main described block, is that needed?
```suggestion
    describe(' when data is not loaded', () => {
```

Reply:
done!